aiojobs/_scheduler.py

```diff
@@ -71,8 +81,19 @@ def closed(self):
         if self._closed:
             raise RuntimeError("Scheduling a new job after closing")
         job = Job(coro, self, self._loop)
-        should_start = (self._limit is None or
-                        self.active_count < self._limit)
+        should_start = (self._limit is None
```

Comment:
Please keep existing formatting for the line.
The same should be applied to `should_wait`